src/trunk.c

```diff
@@ -8811,75 +8824,135 @@ trunk_branch_count_num_tuples(trunk_handle *spl,
    }
 }
 
+/*
+ * Stats struct to return trunk node usage statistics via print interface
+ */
+typedef struct trunk_node_stats {
+   platform_log_handle *log_handle;
+   uint64               num_nodes;
+   uint64               num_tuples;
+   uint64               num_kv_bytes;
+} trunk_node_stats;
```

Comment:
New structure added to gather trunk-node specific metrics as we traverse trunk nodes while printing them.